### PR TITLE
Replace quiknode with pokt network endpoint for xdai_mainnet

### PR DESCRIPTION
**Pull request by tolbrino**

### CODE REVIEW

**Comment by SCBuergel** [comment]
@tolbrino what's the rate limit of that one?
I guess major usage requires some token staking, pls ping me on that if / for what / how much we need.

File: src/providers.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ export const providers = {
   matic_rio:
     "https://nd-435-704-707.p2pify.com/9ad58676aefc2aea283ca0a66724bea6",
   xdai_mainnet:
-    "https://still-patient-forest.xdai.quiknode.pro/f0cdbd6455c0b3aea8512fc9e7d161c1c0abf66a/",
+    "https://:842d068fca3c53e26c85c5f4eb49e1fc@poa-xdai.gateway.pokt.network/v1/lb/61113fe218656e00369bdb9b",
```
Comment (0xjjpa) [inline src/providers.ts:7]: I believe there’s an extra colon

**Review by 0xjjpa — CHANGES_REQUESTED** [review]
Please double check whether the extra colon at the beginning is correct

File: src/providers.ts (hunk 1 above)
Comment (tolbrino) [reply src/providers.ts:7]: No. They use an empty HTTP user, no colon would mean the string is the user.

Comment (0xjjpa) [reply src/providers.ts:7]: Gotcha, thanks for clarifying ☺️

**Review by 0xjjpa — APPROVED** [review]
LGTM

**Pull request merged**